static/app/actionCreators/pageFilters.tsx

```diff
@@ -209,6 +220,10 @@ export function initializeUrlState({
   });
 }
 
+function isProjectsValid(projects: ProjectId[]) {
+  return Array.isArray(projects) && projects.every(project => isInteger(project));
```

Comment:
```suggestion
  return Array.isArray(projects) && projects.every(isInteger);
```

Reply:
Will fix in a follow up. I just moved this function